.github/workflows/strain_dispatcher.yml

```diff
@@ -34,7 +34,7 @@ jobs:
               "base_ref": "${{ github.base_ref }}",
               "strain_property": "${{ env.STRAIN_PROPERTY }}",
               "strain_property_value": "${{ env.PROPERTY_VALUE }}",
-              "strain_path": "mango/nelp/strain.yml",
+              "strain_path": "palm/nelp/strain.yml",
```

Comment:
DO you think is better to keep it as a repo var ? like strain_property?
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/7d67fbdc-9470-4ec4-a47a-c841f1787701)

Reply:
Done